### PR TITLE
docs: update current support for built-in completion

### PR DESCRIPTION
**Pull request by MariaSolOs**
A bit of a shameless plug, but when exploring this plugin I noticed that the README says that core doesn't support autocompletion, but as of https://github.com/neovim/neovim/pull/27339 it does :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -857,17 +857,12 @@ To enable inlay hints in Neovim < 0.10, see [this discussion](https://github.com
 
 #### How to enable auto completion?
 
-Neovim *does not* have built-in support for auto-completion,
-and adding support for that is beyond the scope of this plugin.
+As of #ff097f2091e7a970e5b12960683b4dade5563040 Neovim has built-in completion based on the `triggerCharacters` sent by
```
Comment (mrcjkb) [inline README.md:860]:
```suggestion
As of [#ff097f2091e7a970e5b12960683b4dade5563040](https://github.com/neovim/neovim/pull/27339),
Neovim has built-in completion based on the `triggerCharacters` sent by
```

**Review by mrcjkb — APPROVED** [review]
Awesome! :smile: 

And thanks :pray:

File: README.md [hunk 2]
```diff
@@ -857,17 +857,13 @@ To enable inlay hints in Neovim < 0.10, see [this discussion](https://github.com
 
 #### How to enable auto completion?
 
-Neovim *does not* have built-in support for auto-completion,
-and adding support for that is beyond the scope of this plugin.
+As of [#ff097f2091e7a970e5b12960683b4dade5563040](https://github.com/neovim/neovim/pull/27339),
+Neovim has built-in completion based on the `triggerCharacters` sent by
+language servers. Omni completion is also available for a more traditional `vim`-like completion experience.
```
Comment (mrcjkb) [inline README.md:862]: style(markdownlint): line length

```suggestion
language servers.
Omni completion is also available for a more traditional `vim`-like completion experience.
```

File: README.md [hunk 3]
```diff
@@ -857,17 +857,14 @@ To enable inlay hints in Neovim < 0.10, see [this discussion](https://github.com
...
-  This plugin will automatically register the necessary client capabilities
-  if you have `cmp-nvim-lsp` installed.
+As of [#ff097f2091e7a970e5b12960683b4dade5563040](https://github.com/neovim/neovim/pull/27339),
+Neovim has built-in completion based on the `triggerCharacters` sent by
+language servers.
+Omni completion is also available for a more traditional `vim`-like completion experience.
+
+For more extensible and complex autocompletion setups you need a plugin such as [`nvim-cmp`](https://github.com/hrsh7th/nvim-cmp)
+and a LSP completion source like [`cmp-nvim-lsp`](https://github.com/hrsh7th/cmp-nvim-lsp).
+This plugin will automatically register the necessary client capabilities if you have `cmp-nvim-lsp` installed.
```
Comment (mrcjkb) [inline README.md:867]: style(markdownlint): line length

```suggestion
This plugin will automatically register the necessary client capabilities
if you have `cmp-nvim-lsp` installed.
```

**Comment by MariaSolOs** [comment]
> Awesome! 😄
> 
> And thanks 🙏

Of course! And sorry for all the issues with the markdown linter :/

**Comment by mrcjkb** [comment]
> And sorry for all the issues with the markdown linter :/

No problem, it's not used my many projects :smile:

**Pull request merged**